### ISSUE TITLE
How to solve ./run.sh: line 120: mongo: command not found

### ISSUE THREAD

**Issue by pty-coding-adventure** [free5gc/free5gc#595]
Hello!
When I tested UERANSIM against free5GC, I found an error in UERANSIM
 [nas] [error] PDU Session Establishment Reject received [INSUFFICIENT_RESOURCES_FOR_SPECIFIC_SLICE_AND_DNN]
And I looked the log in Free5gc and found another error.
log in free5gc:
ubuntu@Free5GC:~/free5gc$  ./force_kill.sh
[sudo] password for ubuntu: 
nrf: no process found
amf: no process found
smf: no process found
udr: no process found
pcf: no process found
udm: no process found
nssf: no process found
ausf: no process found
n3iwf: no process found
upf: no process found
chf: no process found
tcpdump: no process found
Cannot find device "upfgtp"
Cannot find device "ipsec0"
rm: cannot remove '/dev/mqueue/*': No such file or directory
./force_kill.sh: line 22: mongo: command not found
true

ubuntu@Free5GC:~/free5gc$  ./run.sh
log path: ./log/20240812_195138/
2024-08-12T19:51:38.456768664+08:00 [INFO][UPF][Main] UPF version:  
	free5GC version: v3.4.1
	build time:      2024-07-18T07:53:50Z
	commit hash:     705e50db
	commit time:     2024-03-27T16:48:24Z
	go version:      go1.21.8 linux/amd64
2024-08-12T19:51:38.456840252+08:00 [INFO][UPF][CFG] Read config from [./config/upfcfg.yaml]
2024-08-12T19:51:38.457157307+08:00 [INFO][UPF][CFG] ==================================================
2024-08-12T19:51:38.457197150+08:00 [INFO][UPF][CFG] (*factory.Config)(0xc0004c7f40)({
	Version: (string) (len=5) "1.0.3",
	Description: (string) (len=31) "UPF initial local configuration",
	Pfcp: (*factory.Pfcp)(0xc0005895c0)({
		Addr: (string) (len=9) "127.0.0.8",
		NodeID: (string) (len=9) "127.0.0.8",
		RetransTimeout: (time.Duration) 1s,
		MaxRetrans: (uint8) 3
	}),
	Gtpu: (*factory.Gtpu)(0xc000589770)({
		Forwarder: (string) (len=5) "gtp5g",
		IfList: ([]factory.IfInfo) (len=1 cap=1) {
			(factory.IfInfo) {
				Addr: (string) (len=14) "192.168.56.107",
				Type: (string) (len=2) "N3",
				Name: (string) "",
				IfName: (string) "",
				MTU: (uint32) 0
			}
		}
	}),
	DnnList: ([]factory.DnnList) (len=1 cap=1) {
		(factory.DnnList) {
			Dnn: (string) (len=8) "internet",
			Cidr: (string) (len=12) "10.60.0.0/24",
			NatIfName: (string) ""
		}
	},
	Logger: (*factory.Logger)(0xc00009eee0)({
		Enable: (bool) true,
		Level: (string) (len=4) "info",
		ReportCaller: (bool) false
	})
})
2024-08-12T19:51:38.457269050+08:00 [INFO][UPF][CFG] ==================================================
2024-08-12T19:51:38.457281686+08:00 [INFO][UPF][Main] Log level is set to [info]
2024-08-12T19:51:38.457289841+08:00 [INFO][UPF][Main] Report Caller is set to [false]
2024-08-12T19:51:38.457310649+08:00 [INFO][UPF][Main] starting Gtpu Forwarder [gtp5g]
2024-08-12T19:51:38.457329146+08:00 [INFO][UPF][Main] GTP Address: "192.168.56.107:2152"
2024-08-12T19:51:38.459313873+08:00 [ERRO][UPF][Main] UPF Cli Run Error: open Gtp5g: open link: create: operation not supported
./run.sh: line 120: mongo: command not found
true
Started nrf with PID 22214
I'm sure to install mongoDB, what should I do?

**Comment by TYuan0816** [comment]
Hello,

The error `[ERRO][UPF][Main] UPF Cli Run Error: open Gtp5g: open link: create: operation not supported` indicates that the `gtp5g` kernel module is not installed or not correctly loaded on your system.

To resolve this issue, you need to install the `gtp5g` kernel module. You can find the installation steps in the Free5GC guide here: [Free5GC Installation Guide - Install Control Plane Elements](https://free5gc.org/guide/3-install-free5gc/#b-install-control-plane-elements).

Additionally, there's a similar issue discussed on the Free5GC forum, which might provide more context: [Free5GC Forum - UPF Cli Run Error](https://forum.free5gc.org/t/erro-upf-main-upf-cli-run-error-open-gtp5g-open-link-create-operation-not-supported/1795).

Please follow the installation steps, and if the issue persists, feel free to reach out again.

**Comment by pty-coding-adventure** [comment]
@TYuan0816 Thank you very much for your help! The discussion in the second link resolved the issue I was facing. Following the instructions there, I was able to successfully load the gtp5g module and the UPF error no longer occurs.

However, I noticed that after my free5gc VM shutdown, the gtp5g module seems to require reloading. Is this normal behavior? Is there a way to configure it to automatically load during system startup, so I don't have to do it manually each time?

Thank you again for your help. If you have any suggestions on how to set up automatic loading of the gtp5g module, I would greatly appreciate it.

**Comment by pty-coding-adventure** [comment]
@TYuan0816 Thank you very much for your help! The discussion in the second link resolved the issue I was facing. Following the instructions there, I was able to successfully load the gtp5g module and the UPF error no longer occurs.

However, I noticed that after my free5gc VM shutdown, the gtp5g module seems to require reloading. Is this normal behavior? Is there a way to configure it to automatically load during system startup, so I don't have to do it manually each time?

Thank you again for your assistance. If you have any suggestions on how to set up automatic loading of the gtp5g module, I would greatly appreciate it.
